### DISCUSSION TITLE
Release Instance manager

### DISCUSSION THREAD

**Discussion by FaraSys** [longhorn/longhorn#6380] [Q&A] [answered]
Hi everyone!
After upgrade Longhorn 1.4.1 to 1.5.1, How to release Instance manager?
![image](https://github.com/longhorn/longhorn/assets/67997008/053ace02-833e-46f3-b652-9b89c6ef22b4)

**Comment by mantissahz** [comment, marked as answer]
You could scale down the workload to detach the volumes that still use the engine-image v1.4.1, `instance-manager-e` would be deleted and then you could scale up the workload and Longhorn will upgrade volume engine images automatically.

**Reply by FaraSys** [reply]
Thank you!

**Reply by derekbit** [reply]
You can also refer to the [doc](https://longhorn.io/kb/troubleshooting-some-old-instance-manager-pods-are-still-running-after-upgrade/) for the mutilple instance-manager-r/e pods.
